dashboard/src/app/dashboard/dashboard-panel/dashboard-panel.directive.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name dashboard.directive:DashboardPanel
+ * @description This class is handling the directive of the panel of last used entries in the dashboard.
```

Comment:
"This class is handling the directive of the panel of last used entries in the dashboard."
it can be simple dashboard panel for displaying dashboard entries, not only last used. Example - credit card info, used RAM